cdap-docs/admin-manual/incompatibilities.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2019 Cask Data, Inc.
+
+.. _admin-manual-incompatibilities:
+
+=============================
+Incompatibility with CDAP 5.x
+=============================
+
+CDAP 6.0.0 has several incompatible changes, because of that CDAP 6.0.0 does not support upgrade or migration from
```

Comment:
change to:
CDAP 6.0.0 has several incompatible changes. Due to this, CDAP 6.0.0 does not support upgrade or migration from..